### PR TITLE
Duality conversion properties

### PR DESCRIPTION
**Pull request by JDemler**
Addresses Issue #157

### CODE REVIEW

File: src/Categories/Category/Duality.agda [hunk 1]
```diff
@@ -58,3 +58,13 @@ FinitelyCocomplete⇒coFinitelyComplete FC = record
   ; equalizer = λ f g → Coequalizer⇒coEqualizer C (coequalizer f g)
   }
   where open FinitelyCocomplete FC
+
+
+
+module DualityConversionProperties where
+
+  open import Relation.Binary.PropositionalEquality using (_≡_; refl)
```
[the remark this answers is not in the document]
Comment (JDemler) [reply src/Categories/Category/Duality.agda]: done

**Comment by JacquesCarette** [comment]
As this is WIP, should I not pull it in yet?  I'm fine with lots of little PRs, as long as each one of them contains a self-contained improvement.

**Comment by JDemler** [comment]
I need to clean this PR up a bit, but will restrict it to `Categories.Category.Duality` to keep it small and concise.
What about naming? I liked the `op-involutive` very much, but am not sure about my `coCartesian⇔Cocartesian` idea.

**Comment by JDemler** [comment]
This PR is now cleaned up. If you are ok with the naming, you can merge.

**Pull request merged**